x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActions/Flyout/index.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiPortal,
+  EuiSpacer,
+  EuiText,
+  EuiTitle
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { SettingsSection } from './SettingsSection';
+import { ServiceForm } from '../../../../../shared/ServiceForm';
+
+interface Props {
+  onClose: () => void;
+}
+
+export const Flyout = ({ onClose }: Props) => {
```

Comment:
Something like `CustomActionsFlyout` would make import/filename suggestions a bit more useful (it would be easier to look up).